packages/monaco/src/browser/textmate/monaco-textmate-service.ts

```diff
@@ -73,9 +81,17 @@ export class MonacoTextmateService implements FrontendApplicationContribution {
                     format: 'json',
                     content: '{}'
                 };
-            }
+            },
+            theme: this.monacoThemeRegistry.getTheme(MonacoThemeRegistry.DARK_DEFAULT_THEME)
         });
 
+        this.toDispose.push(this.themeService.onThemeChange( themeChange => {
```

Comment:
Should be `onThemeChange(themeChange => {` instead of `onThemeChange( themeChange => {`.
(Whitespace before the `themeChange`.)